### PR TITLE
fix epoch in tctl

### PR DESCRIPTION
**Pull request by shivanshuraj1333**
fixes epoch number

### CODE REVIEW

File: tctl.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package:
   name: tctl
   version: 1.18.0
-  epoch: 9
+  epoch: 1
```
Comment (kaniini) [inline tctl.yaml:4]: epoch cannot go backwards, e.g. 9 to 1

**Review by kaniini — CHANGES_REQUESTED** [review]

File: tctl.yaml (hunk 1 above)
Comment (shivanshu1333) [reply tctl.yaml:4]: yeah, but somebody merged the PR with epoch 9 by mistake before 9 could be reverted back to 1, should we leave it at 9 now?

Comment (mritunjaysharma394) [reply tctl.yaml:4]: Yeah, I think it went unnoticed here, I think reviewers had left thought but somehow it got merged. https://github.com/wolfi-dev/os/pull/8897/files#r1410816853

Comment (ajayk) [reply tctl.yaml:4]: this is not needed anymore lets close this

**Comment by mamccorm** [comment]
[ajayk](https://github.com/ajayk) [4 days ago](https://github.com/wolfi-dev/os/pull/9794#discussion_r1427524687)
this is not needed anymore lets close this

**Pull request closed without merging**